ignite/engine/__init__.py

```diff
@@ -51,6 +51,7 @@ def supervised_training_step(
     non_blocking: bool = False,
     prepare_batch: Callable = _prepare_batch,
     output_transform: Callable = lambda x, y, y_pred, loss: loss.item(),
+    gradient_accumulation_steps: int = 1,
```

Comment:
L103 defines the usage. IMO we can define this parameter as optional int > 1, default to `None`. What do you think ?